packages/zowe-explorer-ftp-extension/package.json

```diff
@@ -45,8 +45,8 @@
     "vscode": "^1.43.0"
   },
   "dependencies": {
-    "@zowe/zowe-explorer-api": "1.20.0-SNAPSHOT",
     "@zowe/zos-ftp-for-zowe-cli": "1.5.0",
+    "@zowe/zowe-explorer-api": "file:../../dist/zowe-zowe-explorer-api-1.20.0-SNAPSHOT.tgz",
```

Comment:
hmm. I feel like it shouldn't be set-up this way. It will encounter an error if the tgz file is not available in the dist folder.